File: src/ru/org/linux/util/ServletParameterBadValueException.java
package ru.org.linux.util;

public class ServletParameterBadValueException extends ServletParameterException {
  ServletParameterBadValueException(String name, Exception e) {
    super("Bad format of '" + name + "' " + e.toString());
  }

  ServletParameterBadValueException(String name, String info) {
    super("Bad format of '" + name + "' " + info);
  }
}

File: src/main/scala/ru/org/linux/util/ServletParameterBadValueException.scala
/*
 * Copyright 1998-2026 Linux.org.ru
 *    Licensed under the Apache License, Version 2.0 (the "License");
 *    you may not use this file except in compliance with the License.
 *    You may obtain a copy of the License at
 *
 *        http://www.apache.org/licenses/LICENSE-2.0
 *
 *    Unless required by applicable law or agreed to in writing, software
 *    distributed under the License is distributed on an "AS IS" BASIS,
 *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *    See the License for the specific language governing permissions and
 *    limitations under the License.
 */
package ru.org.linux.util

class ServletParameterBadValueException private (message: String) extends ServletParameterException(message):
  private[util] def this(name: String, e: Exception) = this(s"Bad format of '$name' $e")
  def this(name: String, info: String) = this(s"Bad format of '$name' $info")
